source/Ship.h

```diff
@@ -354,10 +354,14 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	double Mass() const;
 	double InertialMass() const;
 	double TurnRate() const;
-	double Acceleration() const;
-	double MaxVelocity() const;
+	double ForwardThrust(bool allThrust) const;
+	double ForwardAcceleration(bool allThrust) const;
 	double ReverseAcceleration() const;
-	double MaxReverseVelocity() const;
+	double Acceleration(bool allThrust) const;
+	bool PrefersForwardAcceleration(bool allThrust) const;
+	double MaxForwardSpeed(bool allThrust) const;
+	double MaxReverseSpeed() const;
+	double MaxSpeed(bool allThrust) const;
```

Comment:
```suggestion
	double Acceleration(bool allThrust = false) const;
	bool PrefersForwardAcceleration(bool allThrust = false) const;
	double MaxForwardSpeed(bool allThrust = false) const;
	double MaxReverseSpeed() const;
	double MaxSpeed(bool allThrust = false) const;
```

false seems to be the most common case, like this it will save some changes

Reply:
obviously also remove false everywhere

Reply:
@RisingLeaf see above